test/unit-test/test-Model.js

```diff
@@ -4491,6 +4491,340 @@ describe('Unit Test/Model Test', function() {
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, new_height, new_width, depth]});
+        assert.throws(() => {
+          model.addOperation(nn.RESIZE_BILINEAR, [0, 1, 2], [3, 4]);
+        });
+      });
+    });
+
+    it('raise error when the length of inputs is less than 3 for "RESIZE_BILINEAR" operation', () => {
+      return nn.createModel(options).then((model) => {
+        let batches = 2;
+        let height = 3;
+        let width = 4;
+        let depth = 5;
+        let new_height = 6;
+        let new_width = 6;
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, height, width, depth]});
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(1, new Int32Array([new_height]));
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(2, new Int32Array([new_width]));
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, new_height, new_width, depth]});
+        assert.throws(() => {
+          model.addOperation(nn.RESIZE_BILINEAR, [0, 1], [2]);
+        });
+      });
+    });
+
+    it('raise error when the length of outputs is 0 not 1 for "RESIZE_BILINEAR" operation', () => {
+      return nn.createModel(options).then((model) => {
+        let batches = 2;
+        let height = 3;
+        let width = 4;
+        let depth = 5;
+        let new_height = 6;
+        let new_width = 6;
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, height, width, depth]});
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(1, new Int32Array([new_height]));
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(2, new Int32Array([new_width]));
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, new_height, new_width, depth]});
+        assert.throws(() => {
+          model.addOperation(nn.RESIZE_BILINEAR, [0, 1, 2], []);
+        });
+      });
+    });
+
+    it('raise error when the length of inputs is 0 not 3 for "RESIZE_BILINEAR" operation', () => {
+      return nn.createModel(options).then((model) => {
+        let batches = 2;
+        let height = 3;
+        let width = 4;
+        let depth = 5;
+        let new_height = 6;
+        let new_width = 6;
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, height, width, depth]});
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(1, new Int32Array([new_height]));
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(2, new Int32Array([new_width]));
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, new_height, new_width, depth]});
+        assert.throws(() => {
+          model.addOperation(nn.RESIZE_BILINEAR, [], [3]);
+        });
+      });
+    });
+
+    it('raise error when the length of inputs and outputs is 0 for "RESIZE_BILINEAR" operation', () => {
```

Comment:
This is duplicated case since you have newed two cases for input length =0 or output length=0